.github/workflows/general_handle-stale-issues-and-prs.yml

```diff
@@ -0,0 +1,34 @@
+name: 'Close stale issues and PRs'
+on:
+  workflow_call:
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      # only needed for delete-branch option (which we are currently not using)
+      # contents: write
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v9
+        with:
+          operations-per-run: 50
+
+          # stale issue handling
+          stale-issue-message: 'This issue is stale because it has been open 7 days with no activity. Remove stale label or comment or this will be closed in another 5 days.'
+          close-issue-message: 'This issue was closed because it has been stalled for 5 days with no activity.'
```

Comment:
```suggestion
          close-issue-message: 'This issue was closed because it has been stale for 5 days with no activity.'
```

Reply:
But that would be incorrect... at that point it would have been 12 days stale, but 5 days stalled

Reply:
But I do agree that this is a bit poorly written. What about this?

"This issue has been inactive for 7 days. Please remove the stale label or leave a comment to keep it open. Otherwise, it will be closed in 5 days."

"This issue has been closed due to 5 days of inactivity after being marked stale."